Quintet-iOS/View/CalendarView.swift

```diff
@@ -0,0 +1,157 @@
+    @StateObject private var viewModel = DateViewModel()
+    @ObservedObject private var coreDataViewModel = CoreDataViewModel()
+    @State private var selectedDate = Date()
+    @State private var isShowPopup = false
+    @State var currentMonth: Int = 0
+    let days: [String] = ["일", "월", "화", "수", "목", "금", "토"]
+    
+    var body: some View {
+        
+        VStack(spacing: 13) {
+            
+            HStack(spacing: -7){
+                
+                Button(action: {
+                    isShowPopup = true
+                }) {
+                    viewModel.yearMonthButtonTextRecordVer
+                        .padding(.horizontal)
+                        .foregroundColor(Color.black)
+                    
+                }.sheet(isPresented: $isShowPopup) {
+                    CalendarYearMonthPickerPopup(viewModel: viewModel, isShowPopup: $isShowPopup)
+                        .frame(width: 300, height: 400)
+                        .background(BackgroundClearView())
+                        .ignoresSafeArea()
+                }
+                
+                Image(systemName: isShowPopup ? "chevron.up" : "chevron.down")
+                    .foregroundColor(.black)
+                    .font(.system(size: 15))
+            }
+            .padding(.bottom, 17)
+            
+            HStack(spacing: 2) {
+                
+                ForEach(days, id: \.self) { day in
+                    
+                    Text(day)
+                        .fontWeight(.medium)
+                        .padding(.horizontal)
+                        .foregroundColor(Color("DarkGray"))
+                    
+                }
+            }
+            
+            Divider()
+                .padding(.horizontal)
+            
+            let columns = Array(repeating: GridItem(.fixed(41)), count: 7)
+            LazyVGrid(columns: columns, spacing: 9) {
+                ForEach(extractDate()) { value in
+                    if value.day != -1 {
+                        CardView(value: value)
+                            .background (
+                                Circle()
+                                    .frame(width: 40, height: 40)
+                                    .foregroundColor(Color("DarkQ"))
+                                    .opacity(isSameDay(date1: value.date, date2: currentDate) ? 1 : 0)
+                            )
+                            .onTapGesture {
+                                currentDate = value.date
+                            }
+                    } else {
+                        Color.clear
+                    }
+                }
+            }
+            .fontWeight(.light)
+        }
+    
+        if let task = coreDataViewModel.getRecordMetaData(selectedYear: viewModel.selectedYear, selectedMonth: viewModel.selectedMonth).first (where: { task in
+                return isSameDay(date1: task.date, date2: currentDate)
+            }) {
```

Comment:
```suggestion
if let task = coreDataViewModel
    .getRecordMetaData(
        selectedYear: viewModel.selectedYear,
        selectedMonth: viewModel.selectedMonth
    )
    .first(where: { task in
        return isSameDay(date1: task.date, date2: currentDate)
    }) {
```
위와 같이 작성하면 가독성이 더 좋아질 것 같습니다!